locale/rma_ch_de_utf/Num2text

```diff
@@ -0,0 +1,185 @@
+#=====================================================================
```

Comment:
Bitte nicht als neue Sprache, wenn dann iso in eine neue sprache packen, ansonstne müssten wir alle User updaten. usw.

Reply:
Denke nicht, dass das geht weil der ledger geht ja auf den Namen im LANGUAGE file, welcher neu sein muss (UTF-8)